Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml

```
@@ -38,16 +38,60 @@
   - type: StaticPrice
     price: 500
 
-- type: entity
+- type: entity #Frontier
   name: Kardashev-Mosin
   parent: BaseWeaponSniper
   id: WeaponSniperMosin
-  description: A civilian grade weapon for hunting, or endless trench warfare. Uses .30 rifle ammo.
+  description: A civilian grade weapon for hunting, or endless trench warfare. Uses .30 rifle ammo. Now with new shiney bayonet!
```

Reads a bit strangely, as if this bayonet is a new device in-universe.  As an alternative, I'd put this forth:

```suggestion
  description: A civilian grade weapon for hunting, or endless trench warfare. Uses .30 rifle ammo, equipped with standard-issue bayonet.
```

If you don't like that, you misspelt shiny.

```suggestion
  description: A civilian grade weapon for hunting, or endless trench warfare. Uses .30 rifle ammo. Now with new shiny bayonet!
```